pkg/cmd/server/api/types.go

```diff
@@ -279,6 +279,21 @@ type MasterConfig struct {
 
 	// NetworkConfig to be passed to the compiled in network plugin
 	NetworkConfig MasterNetworkConfig
+
+	// JenkinsPipelineConfig holds information about the default Jenkins template
+	// used for JenkinsPipeline build strategy.
+	JenkinsPipelineConfig JenkinsPipelineConfig `json:"jenkinsPipelineConfig"`
```

Comment:
Don't mean to be annoying - there's a lot of noise to wade through each day & didn't want this to get missed.